cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -321,6 +321,9 @@ class EfsSettingsSchema(BaseSchema):
     deletion_policy = fields.Str(
         validate=validate.OneOf(DELETION_POLICIES), metadata={"update_policy": UpdatePolicy.SUPPORTED}
     )
+    accesspoint_id = fields.Str(
+        validate=validate.Regexp(r"^fsap-[0-9a-z]{17}$"),
```

Comment:
The property `accesspoint_id` should not support updates, at least for the first iteration on the feature. 
Please add `, metadata={"update_policy": UpdatePolicy.UNSUPPORTED}`.